qcodes/dataset/experiment_container.py

```diff
@@ -333,4 +356,5 @@ def load_or_create_experiment(experiment_name: str,
                                         conn=conn)
         else:
             raise exception
+    _set_active_experiment_id(experiment.exp_id)
```

Comment:
As far as I can see this is not needed. Since both load_experiment_by_name and new_experiment does the same

Reply:
AHHHH. Correct :)
One less active_experiment